RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -43,29 +43,53 @@ public struct CustomerCenterView: View {
 
     private let mode: CustomerCenterPresentationMode
 
+    /// A flag indicating whether the view is already embedded in a navigation stack.
+    ///
+    /// - When set to `true`, the view must be part of an existing NavigationStack / NavigationView.
+    /// - When set to `false`, the view is not part of an external NavigationStack / NavigationView and uses
+    /// one internally.
+    private let isEmbeddedInNavigationStack: Bool
+
     /// Create a view to handle common customer support tasks
     /// - Parameters:
     ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions
     ///   from the Customer Center.
-    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil) {
-        self.init(customerCenterActionHandler: customerCenterActionHandler, mode: .default)
+    ///   - isEmbeddedInNavigationStack: Whether this view is already inside a NavigationStack / NavigationView
+    public init(
+        customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+        isEmbeddedInNavigationStack: Bool = false
+    ) {
+        self.init(
+            customerCenterActionHandler: customerCenterActionHandler,
+            mode: .default,
+            isEmbeddedInNavigationStack: isEmbeddedInNavigationStack
+        )
     }
 
     /// Create a view to handle common customer support tasks
     /// - Parameters:
     ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions
     ///   from the Customer Center.
-    init(customerCenterActionHandler: CustomerCenterActionHandler? = nil,
-         mode: CustomerCenterPresentationMode) {
+    ///   - mode: The presentation mode for the Customer Center
+    ///   - isEmbeddedInNavigationStack: Whether this view is already inside a navigation stack
+    init(
+        customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+        mode: CustomerCenterPresentationMode,
+        isEmbeddedInNavigationStack: Bool = false
+    ) {
         self._viewModel = .init(wrappedValue:
                                     CustomerCenterViewModel(customerCenterActionHandler: customerCenterActionHandler))
         self.mode = mode
+        self.isEmbeddedInNavigationStack = isEmbeddedInNavigationStack
     }
 
-    fileprivate init(viewModel: CustomerCenterViewModel,
-                     mode: CustomerCenterPresentationMode = CustomerCenterPresentationMode.default) {
+    fileprivate init(
+        viewModel: CustomerCenterViewModel,
+        mode: CustomerCenterPresentationMode = CustomerCenterPresentationMode.default
+    ) {
```

Comment:
Btw, do we know if there is some agreed standard on about this kind of formatting?

I see you've changed from 
```
init(param1: Type1,
     param2: Type2)
```

to 

```
init(
     param1: Type1,
     param2: Type2
)
```

Whereas I prefer the former, I don't have any problem with agreeing on the latter. But perhaps we should all stick to the same style?

Reply:
I've reverted this for now 👍